client/index.html

```diff
@@ -293,20 +293,36 @@ <h1 class="title">Settings</h1>
 								</label>
 							</div>
 							{{/if}}
+							{{#unless public}}
+							<div class="col-sm-12">
+								<h2>Push Notifications</h2>
+							</div>
+							<div class="col-sm-12">
+								<button type="button" class="btn" id="pushNotifications" disabled data-text-alternate="Unsubscribe from push notifications">Subscribe to push notifications</button>
```

Comment:
So yeah, as discussed, let's keep UX improvements of this for a latter PR. But it's definitely going to take some time, discussions and attempts before we hit a correct UI/UX.
In the meantime, could you make this an "Enable push notifications" checkbox instead? Thanks! That's my last request before approving this 😊